src/app/components/MediaLoader/index.tsx

```diff
@@ -241,7 +241,8 @@ const MediaLoader = ({ blocks, className, embedded, uniqueId }: Props) => {
 
   const hasPlaceholder = Boolean(showPlaceholder && placeholderSrc);
 
-  const showPortraitTitle = orientation === 'portrait' && !embedded;
+  const showPortraitTitle =
+    orientation === 'portrait' && pageType !== MEDIA_ARTICLE_PAGE && !embedded;
```

Comment:
Nothing to action but seems like disco team will probably update this at some point soon to incorporate `home` pages for the PV work we are doing